primality.py

```diff
@@ -0,0 +1,17 @@
+
+# coding: utf-8
+
+# In[2]:
+
+
+num =int(input())
```

Comment:
User will never understand with showing any text in console.

`num = int(input("Enter a number to check whether it's prime or not "))`